docs/visual-basic/misc/bc36558.md

```diff
@@ -84,5 +84,5 @@ End Namespace
  [NOT IN BUILD: Attributes Overview in Visual Basic](http://msdn.microsoft.com/en-us/0d0cff64-892d-4f57-83bd-bef388553d4f)   
  [NOT IN BUILD: Custom Attributes in Visual Basic](http://msdn.microsoft.com/en-us/d72d8a5c-8f64-4614-b15b-cad66845d047)   
```

Comment:
These two can be replaced with `[Creating custom attributes](~docs/visual-basic/programming-guide/concepts/attributes/creating-custom-attributes.md)`, which should be near the end of the See also section.